src/BlockModifiers.php

```diff
@@ -51,7 +51,7 @@ public function getIterator()
     public function __toString(): string
     {
         $scopeModified = array_reduce($this->modifiers, [$this, 'reduce'], '');
-        $value = preg_replace('/\s{2,}/', ' ', $scopeModified);
+        $value = (string)preg_replace('/\s{2,}/', ' ', $scopeModified);
```

Comment:
Why casting here? Isn't a string already returned?

Reply:
It could be `null`

> If matches are found, the new subject will be returned, otherwise subject will be returned unchanged or NULL if an error occurred.

https://www.php.net/manual/en/function.preg-replace.php